modules/log6x/logging-http-forward-6-2.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/log_collection_forwarding/configuring-log-forwarding.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="logging-http-forward-6-2_{context}"]
+= Forwarding logs over HTTP
+
+To enable forwarding logs over HTTP, specify `http` as the output type in the `ClusterLogForwarder` custom resource (CR).
+
+.Procedure
+
+* Create or edit the `ClusterLogForwarder` CR using the template below:
++
+.Example ClusterLogForwarder CR
+[source,yaml]
+----
+apiVersion: observability.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+  name: <log_forwarder_name> <1>
+  namespace: <log_forwarder_namespace> <2>
+spec:
+  serviceAccountName: <service_account_name> <3>
+  outputs:
+    - name: httpout-app
+      type: http
+      url: <4>
+      http:
+        proxyURL: <proxy_URL> <5>
+        headers: <6>
+          h1: v1
+          h2: v2
+        method: POST
+      secret:
+        name: <7>
+      tls:
+        insecureSkipVerify: <8>
+  pipelines:
+    - name:
```

Comment:
Please provide a name for the pipeline here.

Reply:
updated as per your comment above.